toolkit/imageconfigs/packagelists/core-packages-image-aarch64.json

```diff
@@ -1,6 +1,7 @@
 {
     "packages": [
         "shim-unsigned",
+        "grub2",
```

Comment:
What is the size increase for including grub2 package? We've previously removed packages like grub2 due to concerns on the minimal rootfs size.

Reply:
wc -c < grub2-2.06-11.cm2.x86_64.rpm
10965886  # (~11MB)
wc -c < /sbin/grub2-mkconfig
8279  # (~8KB)

Ideally we could move grub2-mkconfig into grub2-configuration.rpm for leaner packaging. I think it is problematic to move a file between subpackages however...

Reply:
A few options off the top of my head:
1) patch grub-mkconfig to support custom roots so we can just use it from the setup chroot even if the image config doesn't include it.
2) temporarily copy grub-mkconfig from the setup chroot into the install chroot, then clean it up after we are done
3) do a correct sub-package where we can install grub-mkconfig without pulling the rest of grub

Reply:
grub2-mkconfig can also call the following bins:

grub2-file   --> 2.5M
grub2-editenv --> 0.9M
grub2-probe --> 2.7M
grub2-script-check --> 0.7M

Knowing that grub2 ~= 36M, packaging just these tools separately would save our images (36-7.7) 28.3M

Reply:
Update: grub2 is now only required in images whose config opts in to the grub2-mkconfig flow. The tools enforce that any mkconfig-based image has grub2.